### DISCUSSION TITLE
Request to Allow LineageOS UL

### DISCUSSION THREAD

**Discussion by neogen6** [ROM-builders/temporary#24181] [Ideas]
My Manifest(addison-20.0 branch): https://github.com/addison-experimentations/local_manifests/blob/addison-20.0/local_manifest.xml

UL Manifest(lineage-20.0 branch): https://github.com/LineageOS-UL/android/tree/lineage-20.0

Error from testing section: Not supported rom or branch.

PR tag: addison-LineageOS-lineage-20.0-sounddrill

I added LineageOS-UL-lineage-19.1 and LineageOS-UL-lineage-20.0 at the end of line 86

Here is the pull request: https://github.com/ROM-builders/temporary/pull/24182

**Comment by Apon77** [comment]
non popular roms are not added.
